languages/wp-ogdch-theme.pot

```diff
@@ -23,134 +23,154 @@ msgctxt "%s contains path to file."
 msgid "Error locating %s for inclusion"
 msgstr ""
 
-#: 404.php:8 assets/php/theme_functions.php:326
-msgid "Page not found"
```

Comment:
It looks as if a lot of translations where deleted and then a lot of others were readded, when in reality just some translation were changed and most of them stayed the same. Could you change this to commit so that this effect is avoided?

Reply:
Hm, I agree that it's annoying, but the .pot file is generated by a script and I would prefer not to manually fix something that might change the next time we regenerate it. Hope it's okay that I leave it like this.